ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -303,7 +315,10 @@ module.exports = Marionette.LayoutView.extend({
     )
 
     var isEditing = this.$el.hasClass('is-editing')
-    if (isEditing) {
+    if (this.model.attributes.comparator === 'EMPTY') {
+      this.$el.find('filter-comparator').toggle()
+      this.$el.find('filter-input').toggle()
+    } else if (isEditing) {
```

Comment:
Can you help explain to someone who is not proficient with our front end code ( *cough, me*) why we don't need to make the is editing its own if statement instead of an else if? (meaning, why is it okay to skip turn off/on editing now if `this.model.attributes.comparator === 'EMPTY'`)

Reply:
I think the check for isEditing should be in it's own if-else

Reply:
Now, it's worth considering if the 
```else {
      this.turnOffEditing()
    }``` 
needs to be there if the default seems to have editing off